project/src/main/webapp/index.html

```diff
@@ -37,12 +37,44 @@
        <div class="trends" id="trend-2" value="2">Mitch McConnell</div>
        <div class="trends" id="trend-3" value="3">Hand Sanitizer</div>
        <div class="trends" id="trend-4" value="4">Minecraft Nether Update</div>
-       <div id="dots-container">
+       <div id="dot-container">
          <div class="dots" id="dot-1"onclick="updateCurrentSlide(1)"></div>
```

Comment:
Is it worth having these as dot instead of dots too?
(Totally your choice, just a thought)

Reply:
i just thought in terms of english grammar it was more logical